lib/generators/tranzito_utils/templates/tranzito_utils.rb

```diff
@@ -3,6 +3,9 @@
 # Tranzito_utils initializer file
 # Customize only when you want to override the default values of tranzito_utils
 
+# You can update the page title prefix
+# TranzitoUtils::DEFAULT[:title_prefix] = "Name"
```

Comment:
This shouldn't be named `title_prefix` - it's the name of the app. `application_display_name` makes sense to me, or something else that explains what this is great.

I think having it use a default - e.g. `Rails.application.class.module_parent.to_s` would also be good.